### PR TITLE
#468 Add an intermediate class to watch the updates of channels

### PR DESCRIPTION
**Pull request by RkAiman**

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/channelIntre.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/channelIntre.tsx:1]: You have an issue in the file name "channelIntre".
Can you change it to "ChannelIntermediate" (and note the uppercase on first letter)

File: twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsWorkspace/WorkspaceChannels.tsx [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-import { ChannelResource, ChannelType } from 'app/models/Channel';
+import ChannelInter from '../Parts/Channel/channelIntre';
+
+import ChannelWorkspaceEditor from 'scenes/Client/ChannelsBar/Modals/ChannelWorkspaceEditor';
+import WorkspaceChannelList from 'scenes/Client/ChannelsBar/Modals/WorkspaceChannelList';
+
 import Menu from 'components/Menus/Menu.js';
-import Icon from 'app/components/Icon/Icon';
-import { Collection } from 'app/services/CollectionsReact/Collections';
+import Icon from 'components/Icon/Icon';
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsWorkspace/WorkspaceChannels.tsx:17]: I can't see it from Github, but please ensure that there is no unused imports :)

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Just change the file name and we're ready to go !

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/channelIntre.tsx (hunk 1 above)
Comment (RkAiman) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/channelIntre.tsx]: Done

File: twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsWorkspace/WorkspaceChannels.tsx (hunk 2 above)
Comment (RkAiman) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsWorkspace/WorkspaceChannels.tsx:17]: Done

**Pull request merged**